docs/tutorials/fitting_procedure.ipynb

```diff
@@ -97,7 +97,7 @@
    "metadata": {},
    "source": [
     "Before fitting the radial brightness profile, we need to determine the disc's geometry (inclination, position angle and phase center) in order to deproject the visibilities (recall that frank, fitting in 1D, assumes an axisymmetric source).\n",
-    "To do this we pass an object specifying how the geometry is determined to the [FrankFitter](../py_API.rst#frank.radial_fitters.FrankFitter) class.\n",
+    "To do this we pass an object specifying how the geometry is determined to the [FrankFitter](../py_API.rst#frank.radial_fitters.FrankFitter) class. **The position angle and right ascension offset are defined east of north.**\n",
```

Comment:
`and right ascension offset are defined east of north` this is not clear. see my previous comments

Reply:
Fixed in bbda275